---a/net/minecraft/item/ItemAnvilBlock.java
+++b/net/minecraft/item/ItemAnvilBlock.java
@@ -2,18 +2,16 @@
 
 import net.minecraft.block.Block;
 
-public class ItemAnvilBlock extends ItemMultiTexture
-{
+public class ItemAnvilBlock extends ItemMultiTexture {
+
     private static final String __OBFID = "CL_00001764";
 
-    public ItemAnvilBlock(Block block)
-    {
-        super(block, block, new String[] {"intact", "slightlyDamaged", "veryDamaged"});
+    public ItemAnvilBlock(Block block) {
+        super(block, block, new String[]{"intact", "slightlyDamaged", "veryDamaged"});
     }
 
-    public int getMetadata(int damage)
-    {
+    public int getMetadata(int damage) {
         return damage << 2;
     }
-}
+}
